src/apprt/gtk/Window.zig

```diff
@@ -87,37 +83,28 @@ pub fn init(self: *Window, app: *App) !void {
     };
 
     // Create the window
-    const window: *c.GtkWidget = window: {
-        if ((comptime adwaita.versionAtLeast(0, 0, 0)) and adwaita.enabled(&self.app.config)) {
-            const window = c.adw_application_window_new(app.app);
-            c.gtk_widget_add_css_class(@ptrCast(window), "adw");
-            break :window window;
-        } else {
-            const window = c.gtk_application_window_new(app.app);
-            c.gtk_widget_add_css_class(@ptrCast(window), "gtk");
-            break :window window;
-        }
-    };
-    errdefer c.gtk_window_destroy(@ptrCast(window));
+    const gtk_widget = c.adw_application_window_new(app.app);
+    errdefer c.gtk_window_destroy(@ptrCast(gtk_widget));
 
-    const gtk_window: *c.GtkWindow = @ptrCast(window);
-    self.window = gtk_window;
-    c.gtk_window_set_title(gtk_window, "Ghostty");
-    c.gtk_window_set_default_size(gtk_window, 1000, 600);
-    c.gtk_widget_add_css_class(@ptrCast(gtk_window), "window");
-    c.gtk_widget_add_css_class(@ptrCast(gtk_window), "terminal-window");
+    self.window = @ptrCast(@alignCast(gtk_widget));
+
+    c.gtk_window_set_title(self.window, "Ghostty");
+    c.gtk_window_set_default_size(self.window, 1000, 600);
+    c.gtk_widget_add_css_class(gtk_widget, "adw");
```

Comment:
I think we should remove this CSS class.

Reply:
Normally I'd say for the sake of backwards compatibility we should just keep it since it costs nothing, but I don't think anyone is actively relying on it and the fix is simple enough (remove the selector).

Reply:
Done.